components/rhel/rhel-ose/Vagrantfile

```diff
@@ -5,6 +5,9 @@
 # This variable is ignored for Hyper-V provider.
 PUBLIC_ADDRESS="10.1.2.2"
 
+#Image tag for OCP
+IMAGE_TAG="v3.3.1.3"
```

Comment:
@praveenkumar we can not comment this as we are using the variable in the Vagrantfile.